eng/docker/libraries-sdk.linux.Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Builds and copies library artifacts into target dotnet sdk image
-ARG BUILD_BASE_IMAGE=mcr.microsoft.com/dotnet-buildtools/prereqs:centos-7-f39df28-20191023143754
+ARG BUILD_BASE_IMAGE=mcr.microsoft.com/dotnet-buildtools/prereqs:centos-7
```

Comment:
8.0 will not support Centos 7 AFAIK (hurray!) we should update this at some point

Reply:
Stream9??? Or should we swap for different distro?

Reply:
the `centos-7` images have build tools and we use them for product. I don't know if that is needed. 
I think we should pick whatever is easiest for us to maintain.

Reply:
I'm switching to `centos-stream8` in #85342, which seems to work.